tests/test_replay_buffer.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+from stable_baselines.deepq.replay_buffer import ReplayBuffer, PrioritizedReplayBuffer
+
+
+def test_extend_uniform():
+    nvals = 16
+    states = [np.random.rand(2, 2) for _ in range(nvals)]
+    actions = [np.random.rand(2) for _ in range(nvals)]
+    rewards = [np.random.rand() for _ in range(nvals)]
+    newstate = [np.random.rand(2, 2) for _ in range(nvals)]
+    done = [np.random.randint(0, 2) for _ in range(nvals)]
+
+    size = 32
+    baseline = ReplayBuffer(size)
+    ext = ReplayBuffer(size)
+    for data in zip(states, actions, rewards, newstate, done):
+        baseline.add(*data)
+
+    states, actions, rewards, newstates, done = map(
+        np.array, [states, actions, rewards, newstate, done])
+    
+    ext.extend(states, actions, rewards, newstates, done)
+    assert len(baseline) == len(ext)
+    
+    # assert values
```

Comment:
Nitpick: For clarity this could say "Check the buffers have same contents". Same below.